package.json

```diff
@@ -29,12 +29,15 @@
     "@emotion/styled": "^11",
     "@react-oauth/google": "^0.2.6",
     "@reduxjs/toolkit": "^1.8.2",
+    "aws-sdk": "^2.1231.0",
     "axios": "^0.27.2",
+    "dotenv": "^16.0.3",
```

Comment:
Why do we need dotenv? I did not find anywhere you use it.

Reply:
Remnants of previous attempts at different methods, deleted